app/src/main/java/org/p2p/wallet/utils/CoroutinesExtensions.kt

```diff
@@ -21,7 +21,8 @@ suspend fun <T> retryRequest(
         try {
             return block()
         } catch (e: IOException) {
-            Timber.tag("Coroutines").d("Retrying request since it failed")
+            Timber.tag("Coroutines").i(e)
+            Timber.tag("Coroutines").i("Retrying request since it failed")
```

Comment:
why not join the 2 logs into one?
`Timber.tag("Coroutines").i(e, "Retrying request since it failed")`